frontend/tsconfig.json

```diff
@@ -17,5 +17,6 @@
         "noEmit": true,
         "jsx": "react-jsx"
     },
-    "include": ["src", "@types/dom.d.ts"]
+    "include": ["src", "@types/dom.d.ts"],
+    "exclude": ["node_modules"]
```

Comment:
I'm assuming this is to silence some warnings or errors caused by libraries?

Reply:
I created the augmentation of the CustomEvents but sometimes imports did not find my type definitions.
I would need to experiment more to see if this line is actually 100% needed, but after these changes I did not see the type definitions not found error anymore.